client/gradle/dependencies.gradle

```diff
@@ -10,6 +10,7 @@ dependencies {
   implementation group: "org.glassfish.jersey.media", name: "jersey-media-json-jackson", version: jerseyVersion
   implementation group: "org.glassfish.jersey.media", name: "jersey-media-multipart", version: jerseyVersion
   implementation group: "org.glassfish.jersey.connectors", name: "jersey-jdk-connector", version:jerseyVersion
+  implementation group: 'org.glassfish.jersey.inject', name: 'jersey-hk2', version: jerseyVersion
```

Comment:
I _think_ for this to be consumed by users of the jar, it should be `application` instead of `implementation`.

Reply:
Ah, this helped point me to https://docs.gradle.org/current/userguide/library_vs_application.html). Looks like the keyword is `api`.